pkg/store/postings_codec.go

```diff
@@ -33,8 +33,9 @@ func isDiffVarintSnappyEncodedPostings(input []byte) bool {
 // diffVarintSnappyEncode encodes postings into diff+varint representation,
 // and applies snappy compression on the result.
 // Returned byte slice starts with codecHeaderSnappy header.
-func diffVarintSnappyEncode(p index.Postings) ([]byte, error) {
-	buf, err := diffVarintEncodeNoHeader(p)
+// Values argument is expected number of postings, used for preallocating buffer.
+func diffVarintSnappyEncode(p index.Postings, values int) ([]byte, error) {
```

Comment:
Maybe renaming `values` in `length` could be a bit more clear.

Reply:
:+1:

Reply:
ok, `length` it is.